calibrators/marker_radar_lidar_calibrator/README.md

```diff
@@ -0,0 +1,141 @@
+# marker_radar_lidar_calibrator
+
+A tutorial for this calibrator can be found [here](../docs/tutorials/marker_radar_lidar_calibrator.md)
+
+## Purpose
+
+The package `marker_radar_lidar_calibrator` allows extrinsic calibration between radar and 3d lidar sensors used in autonomous driving and robotics.
+
+Currently, the calibrator only supports the radar that includes distance and azimuth angle, but without elevation angle. For example, ARS 408 radar can be calibrated with this tool. Also, note that the 3d lidar should have a resolution that is high enough to scan several points on the radar reflector (calibration target).
+
+## Inner-workings / Algorithms
+
+The calibrator is designed to accurately predict the transformation between radar and lidar sensors. It starts by pinpointing the central points of reflectors within lidar pointclouds and radar messages, then aligns these points for precise matching. An SVD-based and a yaw-only rotation estimation algorithm are applied to these correlated points to determine the transformation. Specifically, the calibration process consists of four primary steps: constructing a background model, extracting the foreground to detect reflectors, matching and filtering lidar and radar detections, and finally executing the calibration.
+
+### Step 1: Background model construction
+
+Firstly, given the challenge of reliably detecting reflectors, background models for both lidar and radar are constructed from the lidar pointcloud and radar message within a user-defined calibration area, which lacks any calibration targets (such as radar reflectors). More specifically, these background models consist of uniform binary voxel grids that denote whether each voxel represents the background.
```

Comment:
Throughout the documment, there are several mentions to radar messages. Messages is a ROS jargon (in this case). Depending on what you want to say, they are radar detections, objects, etc (i.e., not use radar messages)

Reply:
There is no mention in this section of why background models are related to " the challenge of reliably detecting reflectors"

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/23bce5cf65091f2dacc27565fbfbc32816facc8a.
The drawing is uploaded as well (it was fixed by several commits so might be easier to look at the new page)

Reply:
Thanks for making a new figure. While quite pretty, it does not convey the nature of the problem.
The reflector themselves should face the radar and that fact should be specified wherever applicable in the document

Reply:
fixed in  https://github.com/tier4/CalibrationTools/pull/164/commits/0c8965d4b948202bfb0dac9df864c970078a558d